packages/cli-e2e/utils/login.ts

```diff
@@ -79,7 +79,7 @@ export function runLoginCommand(orgId: string, env: string) {
     .do(answerPrompt('n'))
     .once();
 
-  return loginTerminal;
+  return loginTerminal.when('exit').on('process').do().once();
```

Comment:
Funny thing: the E2E Login process was existing just after setting `accessToken` and was not waiting for the other configs to be set, notably the orgId.